### PR TITLE
pre-commit: enable Python Flake8 rule `E713`

### PR DESCRIPTION
**Pull request by jbampton**
https://www.flake8rules.com/rules/E713.html

### Description

This PR adds another check or test to our Python linting framework.

Sets a consistent code standard and stops regressions.

"Test for membership should be 'not in' (E713)"

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [X] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [ ] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [ ] Trivial

### How Has This Been Tested?

Ran:

`pre-commit run --all-files`

### CODE REVIEW

**Review by DaanHoogland — APPROVED** [review]
clgtm, needs testing

**Comment by DaanHoogland** [comment]
@blueorangutan package

File: scripts/vm/network/security_group.py [hunk 1]
```diff
@@ -1102,7 +1102,7 @@ def add_network_rules(vm_name, vm_id, vm_ip, vm_ip6, signature, seqno, vmMac, ru
 
         changes = check_rule_log_for_vm(vmName, vm_id, vm_ip, domId, signature, seqno)
 
-        if not 1 in changes:
+        if False in changes:
```
Comment (vishesh92) [inline scripts/vm/network/security_group.py:1105]:
```suggestion
        if 1 not in changes:
```

**Comment by jbampton** [comment]
@blueorangutan package

**Review by weizhouapache — APPROVED** [review]
code lgtm

not tested

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by jbampton** [comment]
@blueorangutan package

**Pull request merged**